gui-apps/gtkgreet/gtkgreet-0.6.ebuild

```diff
@@ -0,0 +1,37 @@
+# Copyright 2019-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="gtk based greeter for greetd"
+HOMEPAGE="https://git.sr.ht/~kennylevinsen/gtkgreet"
+
+inherit meson
+
+if [[ ${PV} == 9999 ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://git.sr.ht/~kennylevinsen/gtkgreet.git"
+else
+	SRC_URI="https://git.sr.ht/~kennylevinsen/gtkgreet/archive/${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64 ~x86"
+fi
+
+LICENSE="GPL-3"
+SLOT="0"
+IUSE="+layershell +man"
+
+RDEPEND="
+	gui-libs/greetd
+"
+BDEPEND="
+	virtual/pkgconfig
+	man? ( app-text/scdoc )
+"
+
```

Comment:
gtk based, but does not depend on gtk?
sounds fishy.

Reply:
Oooof, had totally forgotten to add them.
Added now :smile_cat: